### ISSUE TITLE
How to tune the parameters. AutoTune?

### ISSUE THREAD

**Issue by leifnel** [r-downing/AutoPID#8]
Can the PID tune itself?

Are there any hints?

I've got a thermobox, with a pelter element I can control the heat/cool from -1024 to 1024 with pwm.
An oscilation time of around 5 minutes and an overshoot of 1.5C if I go from full heat to full cool around the setpoint (unrealistic, but to find the parameters of the system)

**Comment by tartavull** [comment]
+1

**Comment by PalantirWielder** [comment]
No, I don't believe this library contains autotuning features. There do appear to be other PID libraries that can do that though.

**Comment by iezqrom** [comment]
which ones?

**Comment by jessedmatlock** [comment]
+1
